scripts/create_topological_sort_of_all_services.py

```diff
@@ -105,22 +105,18 @@ def make_graph() -> Tuple[Dict[str, List[str]], Set[str]]:
                     parsed_script = esprima.parseScript(dep_lines, comment=True)
                     parsed_nodes = parsed_script.body
                     for parsed_node in parsed_nodes:
-                        # For require statements.
-                        if parsed_node.type == 'ExpressionStatement' and (
```

Comment:
What is the rationale for deleting this "if"?

If you leave comments for the reviewer to explain these things prior to review, you'll save a review cycle.

Reply:
Done